src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,34 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = super.factory.openSession();
```

Comment:
```suggestion
            session = factory.openSession();
```
ти маєш напряму доступ до поля так як воно має protected модифікатор